deb/common/docker-ce.docker.service

```
@@ -0,0 +1 @@
+engine/contrib/init/systemd/docker.service
```

This symlink ends up in `debian/` so needs to be `../` prefixed, right? :eyes:

```suggestion
../engine/contrib/init/systemd/docker.service
```

---

Curious that the CI builds succeeded; I bet they're missing systemd service files :grimacing:

---

oh, interesting; so this was a straight revert of your original one; wondering if things were broken there, or is it because we moved things?

---

Oh, it was probably broken there!! :facepalm:

---

I haven't checked yet 🙈 

I'm always confused with these paths as well, because it's possible we put the directories in a different path during build (which I need to check)

---

Yeah, all these files end up under `debian/`, of which `engine/` is a "sibling directory" :sweat_smile: